cacheops/query.py

```diff
@@ -269,9 +270,9 @@ def iterator(self):
             cache_data, ttl = redis_client.get_with_ttl(cache_key)
             cache_read.send(
                 sender=self.model,
-                func=self._cacheprofile['name'],
+                func=self._cacheconf['name'],
```

Comment:
Seems like we are storing the cache profile in two locations, one memoized in the _cacheprofile property, and another more explicitly in _cachconf.